packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -33,8 +34,17 @@ selfUtils.getSelves = (oldSelf, newSelf) => {
   };
 };
 
+selfUtils.getId = (self) => {
+  const person = self.person;
+  let id = null;
+  if (person) {
+    id = person.id;
+  }
+  return id;
+};
+
 selfUtils.getMuted = (self) => {
-  const controls = {self};
+  const controls = self.controls;
```

Comment:
any reason not to destructure?

Reply:
it was giving the wrong object, and didn't look back to it, I was destructuring it incorrectly